template/manifests/Service/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{.APPNAME}}
+  labels:
+    app.kubernetes.io/name: {{.APPNAME}}
+    app.kubernetes.io/part-of: {{.PARTOF}}
+    kubernetes.azure.com/generator: {{.GENERATORLABEL}}
+spec:
+  type: ClusterIP
```

Comment:
should type be configurable?

Reply:
if type is configurable annotations maybe should be too for lb annotations (private, ip, etc)

Reply:
you mean allowing passing a string>string map of annotations as an input and then appending them in the template right?

Reply:
i don't know what the draft format should be but if type is configurable being able to pass in annotations in some form is really useful

Reply:
I think having that configuration will come a little later, this was just meant to be a simple clusterIP service to begin with!